### DISCUSSION TITLE
Support buildpacks

### DISCUSSION THREAD

**Discussion by aalmiray** [jreleaser/jreleaser#1338] [Ideas]
There are two potential integrations: internal & external.

### Internal

Create a JReleaser buildpack. Its job is to take binaries from the previous buildpack step and run JReleaser commands. Once finished it passes control to the next buildpack.

### External

JReleaser calls into buildpacks (as a command line tool) such that given a set of existing binaries it create container images, potentially publishing to container registries as well. JReleaser supports binary distributions at the moment, there may be an option to support source distributions in the future.

**Comment by dashaun** [comment]
The Internal route is absolutely possible.

**Reply by aalmiray** [reply]
I'm a total n00b when it comes to buildpacks. The good news is that it's easy to download and integrate JReleaser on any given environment. Would appreciate any tips to get started with buildpacks and build my own pack.

**Reply by dashaun** [reply]
Would it be possible to get [30-minutes on a zoom](https://calendly.com/dashaun)?  So I can ask you a few questions?
